test/EleventyServeTest.js

```diff
@@ -29,6 +30,7 @@ test("Get Options", t => {
     server: {
       baseDir: "_site"
     },
+    middleware: options.middleware,
```

Comment:
This is a little jank. ava's `deepEqual` compares functions by identity, and there's no way to get two copies of the same function in this scenario since the middleware always returns a new function. So, I'm left with "opting out" of the deep compare here.

Reply:
Hm … you could cast a function to a string (via `+ ''`) and compare the function body :-D

Reply:
Yeah but the “actual” object will still have a function there. I don’t think I can make ava cast that, right?

Reply:
I wouldn't know how :-/